lib/bracket-matcher.coffee

```diff
@@ -145,7 +151,7 @@ class BracketMatcher
     bracketsRemoved
 
   wrapSelectionInBrackets: (bracket) ->
-    return false unless atom.config.get('bracket-matcher.wrapSelectionsInBrackets')
+    return false unless @getScopedSetting('bracket-matcher.wrapSelectionsInBrackets')
 
     if bracket is '#'
```

Comment:
`wrapSelectionInBrackets` is called right after `toggleQuotes` in `insertText`, but I'll remove all the `toggleQuotes` anyway and use `atom.config.observe` with `getRootDescriptor`.